### ISSUE TITLE
[Enhancement Request] Show Searched String in new Search Results Window

### ISSUE THREAD

**Issue by TwilightMercy** [ivarbrandt/skin.fentastic#27]
Hi,

Using latest FENtastic `1.0.12b` from test repository.
New search results window is awesome!

I remember that in `1.0.12a` I saw at the top of the screen the text: "search results for XXXXXX" (the searched text string), or something like that.
in `1.0.12b` I don't see it. did it got removed?

Screenshot:

![image](https://github.com/ivarbrandt/skin.fentastic/assets/29895680/1e6ad955-3b7c-4edd-875e-c32164f5d80d)

**Comment by ivarbrandt** [comment]
I closed the issue because you are asking about features that have not been released in the main repo. In the future, if your issue is about an unreleased version, open an issue as normal, then immediately close it. This way I'll be able to see your issue and address it privately. These rules are only for testing versions. A test version of the skin or helper will have a letter next to the version number. All "official" releases won't.

Now as for your issue, it wasn't removed and it is working fine on my end. Did you change anything when you tried `1.0.12a` vs. `1.0.12b`?

**Comment by TwilightMercy** [comment]
No problem, I understand the rules. 

I will test later a fresh install of latest test version and update.

**Comment by ivarbrandt** [comment]
Thanks I appreciate it.

> I will test later a fresh install of latest test version and update.

No need I found the issue. I had set up the displaying of the string with you in mind but I forgot to put the Hebrew part of it in lol. You'll be able to see it if you change the language to English in Kodi.

Btw is that something you would like? The string would appear close to the clock on the upper right hand corner when the language in Kodi is set to Hebrew.
